src/parsePref.ts

```diff
@@ -113,12 +115,10 @@ function setGitCommittish (committish: string | null) {
   if (committish != null && committish.length >= 7 && committish.slice(0, 7) === 'semver:') {
     return {
       gitCommittish: null,
-      gitRange: decodeURIComponent(committish.slice(7)),
+      gitRange: committish.slice(7),
     }
   }
-  return {
-    gitCommittish: committish === '' ? null : committish,
-  }
+  return {gitCommittish: committish}
```

Comment:
if you name the variable gitCommittish, it will be nicer

```typescript
return { gitCommittish }
```

Reply:
I think it's better to keep "committish" as that's used in the two calling of this function.